cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -621,21 +621,43 @@ class HydratorPlusPlusConfigStore {
             if (!this.validateState()) {
               this.emitChange();
             }
+
             // Once the backend properties are fetched for all nodes, fetch their config jsons.
             // This will be used for schema propagation where we import/use a predefined app/open a published pipeline
             // the user should directly click on the last node and see what is the incoming schema
             // without having to open the subsequent nodes.
+            const reqBody = [];
             this.state.__ui__.nodes.forEach( n => {
               // This could happen when the user doesn't provide an artifact information for a plugin & deploys it
               // using CLI or REST and opens up in UI and clones it. Without this check it will throw a JS error.
-              if (!n.plugin.artifact) { return; }
-              this.HydratorPlusPlusPluginConfigFactory.fetchWidgetJson(
-                n.plugin.artifact.name,
-                n.plugin.artifact.version,
-                n.plugin.artifact.scope,
-                `widgets.${n.plugin.name}-${n.type}`
-              ).then(parseNodeConfig.bind(null, n));
+              if (!n.plugin || !n.plugin.artifact) { return; }
+              const pluginInfo = {
+                name: n.plugin.artifact.name,
+                version: n.plugin.artifact.version,
+                scope: n.plugin.artifact.scope,
+                properties: [
```

Comment:
Any reason why this is an array? We have this as an object with propertyName/value in other places, right? Is that how the backend expects it?

Reply:
this is how the backend expects it